Lib/test/test_xml_etree.py

```diff
@@ -122,6 +128,104 @@ def test_all(self):
         support.check__all__(self, ET, names, blacklist=("HTML_EMPTY",))
 
 
+class ElementTree_Element_UnitTest(unittest.TestCase):
```

Comment:
Why is this not part of the normal Element test class?

Reply:
Which class do you consider the normal Element test class? There are multiple options that might fall under that description.

Which in a way answers your question: I opted to treat all the existing classes as integration tests, and create a separate section for unit tests that test each individual method separately.

Reply:
I meant `BasicElementTest`, seems to fit in there just fine.

Reply:
I was hoping to start a new template for future unit tests, but it you feel strongly about it, I can move them.

Reply:
When working on existing code bases, I think consistency beats purity.